src/commons/pdb.rs

```diff
@@ -0,0 +1,30 @@
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, JsonSchema, PartialEq, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PdbConfig {
+    /// Wether a PodDisruptionBudget should be written out for this role.
+    /// Disabling this enables you to specify you own - custom - one.
+    /// Defaults to true.
+    #[serde(default = "default_pdb_enabled")]
+    pub enabled: bool,
+    /// The number of Pods that are allowed to be down because of voluntary disruptions.
+    /// If you don't explicitly set this, the operator will use a sane default based
+    /// upon knowledge about the individual product.
+    #[serde(default)]
+    pub max_unavailable: Option<u16>,
+}
+
+fn default_pdb_enabled() -> bool {
+    true
+}
```

Comment:
That's quite annoying, but nothing we can do right now. There is a [discussion](https://github.com/serde-rs/serde/issues/368) about this in `serde`, so let's hope there will be support for inline default values soon.

Reply:
Absolutely agreed!